<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClassificationWorkflowWindow</class>
 <widget class="QMainWindow" name="ClassificationWorkflowWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>961</width>
    <height>776</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ilastik Classification Workflow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="VolumeEditorWidget" name="volumeEditorWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>99</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
      <widget class="QWidget" name="widget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>&lt;b&gt;Problem specific labels:&lt;/b&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="LabelListView" name="labelListView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="AddLabelButton">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Add labels</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="SelectFeaturesButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Select Features</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="StartClassificationButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Initialize Workflow</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkInteractive">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Interactive updating</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="interactionComboBox">
          <property name="maxVisibleItems">
           <number>10</number>
          </property>
          <item>
           <property name="text">
            <string>Navigation Mode</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Brushing Mode</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>&lt;b&gt;Layers:&lt;/b&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="LayerWidget" name="layerWidget">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_3" native="true">
             <layout class="QVBoxLayout" name="verticalLayout">
              <property name="margin">
               <number>0</number>
              </property>
              <item>
               <widget class="QToolButton" name="UpButton">
                <property name="toolTip">
                 <string>move layer up</string>
                </property>
                <property name="text">
                 <string>Up</string>
                </property>
                <property name="icon">
                 <iconset>
                  <normaloff>../icons/go-up.png</normaloff>../icons/go-up.png</iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="DownButton">
                <property name="toolTip">
                 <string>move layer down</string>
                </property>
                <property name="text">
                 <string>Down</string>
                </property>
                <property name="icon">
                 <iconset>
                  <normaloff>../icons/go-down.png</normaloff>../icons/go-down.png</iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="DeleteButton">
                <property name="toolTip">
                 <string>delete layer</string>
                </property>
                <property name="text">
                 <string>Delete</string>
                </property>
                <property name="icon">
                 <iconset>
                  <normaloff>../icons/list-remove.png</normaloff>../icons/list-remove.png</iconset>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QToolButton" name="AddLayOutButton">
                <property name="toolTip">
                 <string>add new layer</string>
                </property>
                <property name="text">
                 <string>Add Layout</string>
                </property>
                <property name="icon">
                 <iconset>
                  <normaloff>../icons/list-add.png</normaloff>../icons/list-add.png</iconset>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>961</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionFitToScreen"/>
    <addaction name="actionToggleAllHuds"/>
    <addaction name="actionCenterAllImages"/>
    <addaction name="actionShowDebugPatches"/>
    <addaction name="separator"/>
    <addaction name="actionOnly_for_current_view"/>
    <addaction name="actionFitImage"/>
    <addaction name="actionToggleSelectedHud"/>
    <addaction name="actionCenterImage"/>
    <addaction name="actionOriginalSizeToImage"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionQuit"/>
   <addaction name="actionFitToScreen"/>
   <addaction name="actionShowDebugPatches"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normaloff>../icons/document-open.png</normaloff>../icons/document-open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionFitToScreen">
   <property name="icon">
    <iconset>
     <normaloff>../icons/view-fullscreen.png</normaloff>../icons/view-fullscreen.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Zoom to &amp;fit</string>
   </property>
   <property name="toolTip">
    <string>Fit all</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset>
     <normaloff>../icons/process-stop.png</normaloff>../icons/process-stop.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionShowDebugPatches">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Show Debug Patches</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionToggleAllHuds">
   <property name="text">
    <string>Toggle huds</string>
   </property>
   <property name="toolTip">
    <string>Hide or show all huds</string>
   </property>
  </action>
  <action name="actionOnly_for_current_view">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Only for selected view</string>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionFor_all_axis">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>For all axis</string>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="actionToggleSelectedHud">
   <property name="text">
    <string>Toggle hud</string>
   </property>
   <property name="toolTip">
    <string>toggle selected hud</string>
   </property>
   <property name="shortcut">
    <string>H</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
  </action>
  <action name="actionCenterAllImages">
   <property name="text">
    <string>Center Images</string>
   </property>
  </action>
  <action name="actionCenterImage">
   <property name="text">
    <string>Center Image</string>
   </property>
   <property name="shortcut">
    <string>C</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
  </action>
  <action name="actionFitImage">
   <property name="text">
    <string>Fit image</string>
   </property>
   <property name="toolTip">
    <string>fit image into view</string>
   </property>
   <property name="shortcut">
    <string>F</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionOriginalSizeToImage">
   <property name="text">
    <string>Original size</string>
   </property>
   <property name="toolTip">
    <string>Restore to original size</string>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LabelListView</class>
   <extends>QTableView</extends>
   <header>labelListView</header>
  </customwidget>
  <customwidget>
   <class>LayerWidget</class>
   <extends>QListView</extends>
   <header>volumina.layerwidget.layerwidget</header>
  </customwidget>
  <customwidget>
   <class>VolumeEditorWidget</class>
   <extends>QWidget</extends>
   <header>volumina.volumeEditorWidget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
